apps/ios/Shared/Views/Chat/ReverseList.swift

```diff
@@ -129,8 +132,17 @@ struct ReverseList<Item: Identifiable & Hashable & Sendable, Content: View>: UIV
                     from: nil,
                     for: nil
                 )
+            DispatchQueue.main.async {
+                chatViewScrollState.scrolling = true
+            }
+        }
+
+        override func scrollViewDidScroll(_ scrollView: UIScrollView) {
+            DispatchQueue.main.async {
+                chatViewScrollState.scrolling = false
+            }
```

Comment:
I don't think there is a need to additional dispatch since `ViewController` is already `@MainActor`

Reply:
I got a purple warning from Xcode here or in scrollViewWillBeginDragging, that's why I added main thread call

Reply:
Stanislav open the blockkk!!!

Reply:
Stanislav Dmitrenko open the block!!!